### PR TITLE
Fix PEP8 in fixes

### LINKED ISSUE #878: PEP8 in fixes
dipy/fixes/argparse.py:1:80: E501 line too long (88 > 79 characters)
dipy/fixes/argparse.py:9:80: E501 line too long (80 > 79 characters)
dipy/fixes/argparse.py:17:80: E501 line too long (80 > 79 characters)
dipy/fixes/argparse.py:101:1: E402 module level import not at top of file
dipy/fixes/argparse.py:102:1: E402 module level import not at top of file
dipy/fixes/argparse.py:103:1: E402 module level import not at top of file
dipy/fixes/argparse.py:104:1: E402 module level import not at top of file
dipy/fixes/argparse.py:105:1: E402 module level import not at top of file
dipy/fixes/argparse.py:107:1: E402 module level import not at top of file
dipy/fixes/argparse.py:151:1: E302 expected 2 blank lines, found 1
dipy/fixes/argparse.py:647:44: E127 continuation line over-indented for visual indent
dipy/fixes/argparse.py:700:15: E271 multiple spaces after keyword
dipy/fixes/argparse.py:732:1: E302 expected 2 blank lines, found 1
dipy/fixes/argparse.py:1152:1: E302 expected 2 blank lines, found 1
dipy/fixes/argparse.py:2022:17: E128 continuation line under-indented for visual indent
dipy/fixes/argparse.py:2179:52: E261 at least two spaces before inline comment

### PR DESCRIPTION
**Pull request by ghoshbishakh**
fixes #878

### CODE REVIEW

File: dipy/fixes/argparse.py [hunk 1]
```diff
@@ -1,4 +1,5 @@
-# emacs: -*- coding: utf-8; mode: python; py-indent-offset: 4; indent-tabs-mode: nil -*-
+# emacs: -*- coding: utf-8; mode: python; py-indent-offset: 4;
+# indent-tabs-mode: nil -*-
```
Comment (arokem) [inline dipy/fixes/argparse.py]: Sorry - I believe this needs to all be on one line for emacs to properly pick it up.

**Comment by arokem** [comment]
Nice. Just that one emacs thing. Otherwise - this is good to go!

**Comment by ghoshbishakh** [comment]
@arokem i wonder how this is failing.

**Comment by ghoshbishakh** [comment]
Looks ok to me now.

**Comment by arokem** [comment]
Yep. Looks good to me. If no one objects, I will merge this in a couple of days.

**Pull request merged**